### ISSUE TITLE
safari not working

### ISSUE THREAD

**Issue by gbaumgart** [ibm-js/deliteful#620]
hi,
just updated from 0.4 to lastest these days. I get on Mac and safari (latest) an error in dcl/mini.162  : dcl: wrong super, so the whole scene won't even render. same on iphone of course. 

I use master for most delite ibm packages.

**Comment by gbaumgart** [comment]
chrome and the others : works. Also my whole IDE (maqetta/dcl) based works on safari and safari mobile.

**Comment by gbaumgart** [comment]
found it, had to switch dcl 1.1.2

**Comment by wkeese** [comment]
OK great.  BTW things like dcl will be updated automatically if you do "bower install", but perhaps you have some reason to want to set things up manually.  Anyway, just FYI.

**Comment by gbaumgart** [comment]
Hey, yes of course, I use all from source, like always ;-)

But I just noticed that it still doesn't work iphone - 5s/7.1.1 (many people wont update from here since it ruins the entire phone, like always with apple updates). I deleted all cache folders on the phone and I am really out of advice now. I will strip some server side debugging into it and hope I can fix it. 

It also doesn't work with Chrome on that phone now too, really weird. Using the /sample folders
On iphone - simulator - 5s+ with 9.2 all ok, same for iphone-6s/8.1.

referencing only: https://github.com/uhop/dcl/issues/15.
